Demo/src/main/java/com/paypal/android/usecase/GetBillingAgreementTokenWithoutPurchaseUseCase.kt

```diff
@@ -0,0 +1,52 @@
+package com.paypal.android.usecase
+
+import com.google.gson.JsonObject
+import com.google.gson.JsonParser
+import com.paypal.android.api.services.PayPalApi
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+import javax.inject.Inject
+
+class GetBillingAgreementTokenWithoutPurchaseUseCase
```

Comment:
What is a billing agreement without purchasing? Is this the same as vault without purchase?